endpoints/api/user.py

```diff
@@ -545,6 +546,83 @@ def delete(self):
         return "", 204
 
 
+@resource("/v1/user/initialize")
+class InitializeUser(ApiResource):
+
+    schemas = {
+        "InitializeUser": {
+            "type": "object",
+            "description": "Fields which must be specified for a new user.",
+            "required": [
+                "username",
+                "password",
+            ],
+            "properties": {
+                "username": {
+                    "type": "string",
+                    "description": "The user's username",
+                },
+                "password": {
+                    "type": "string",
+                    "description": "The user's password",
+                },
+                "email": {
+                    "type": "string",
+                    "description": "The user's email address",
+                },
+            },
+        },
+    }
+
+    @verify_not_prod
+    @nickname("initializeUser")
+    @internal_only
+    @validate_json_request("InitializeUser")
+    @anon_allowed
+    def post(self):
+        """
+        Create initial user in an empty database
+        """
+
+        # Ensure that we are using database auth.
+        if app.config["AUTHENTICATION_TYPE"] != "Database":
+            raise request_error(message="Cannot initialize user in a non-database auth system")
+
+        if model.user.get_active_user_count() != 0:
```

Comment:
This is doing a full table count every time someone hits this endpoint. Especially, since anyone can hit this api, nothing prevents from constantly hitting this and exhausting db resources.

Since you don't really care about the actual count, something like `User.select().limit(1)` would be more efficient.